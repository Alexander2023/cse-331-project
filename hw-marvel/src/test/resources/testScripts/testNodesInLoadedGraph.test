# Tests that the nodes in the csv file are added to the graph

LoadGraph small small.csv

ListNodes small