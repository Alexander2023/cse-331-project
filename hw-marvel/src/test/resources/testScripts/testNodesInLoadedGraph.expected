# Tests that the nodes in the csv file are added to the graph

loaded graph small

small contains: c1 c2 c3